phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/UserInterfacePanel.java

```diff
@@ -80,11 +80,11 @@ private void populate() {
         this.showScriptSource.setSelected(javaFxSettingsManager.isViewScriptSource());
         this.showScriptSource.setOnAction(event -> this.save());
 
-        this.showScriptDescription = new Label(tr("Select, if you want to view the source repository of the scripts"));
+        this.showScriptDescription = new Label(tr("View the scripts’ source repository"));
```

Comment:
Shouldn't this be the character `'`?

Reply:
No. Just like in the English quotation marks should not be forced upon different locales.
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table